exporter/exporter.go

```diff
@@ -49,9 +52,30 @@ func New(cfg config.Config) (*Exporter, error) {
 	)
 
 	programInfoDesc := prometheus.NewDesc(
-		prometheus.BuildFQName(prometheusNamespace, "", "ebpf_programs"),
+		prometheus.BuildFQName(prometheusNamespace, "", "ebpf_program_info"),
 		"Info about ebpf programs",
-		[]string{"program", "function", "tag"},
+		[]string{"program", "function", "tag", "id"},
+		nil,
+	)
+
+	programAttachedDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(prometheusNamespace, "", "ebpf_program_attached"),
```

Comment:
Minor nit, I wonder if `ebpf_program` is redundant. Could simplify to just `program`.

Reply:
There's a wrinkle to this: `program` can mean both "ebpf function" (`struct bpf_program` and whatever comes from it) and "a collection of ebpf functions and metrics" (what we call "program" in config).

Internally it's a bit of a mess, but in metrics `program` and `ebpf_program` are different things.

Maybe this is a good time to rename the larger thing to something like "measurement" or "config".

Reply:
Makes sense!